src/text_file_functions.cpp

```diff
@@ -517,7 +517,7 @@ QString AOApplication::get_sfx_suffix(VPath sound_to_check)
 
 QString AOApplication::get_image_suffix(VPath path_to_check, bool static_image)
 {
-  QStringList suffixes { "" };
+  QStringList suffixes {};
```

Comment:
I am a bit concerned that there is some old logic somewhere that uses hardcoded extensions and removing the empty suffix will break it. I suspect that it was caused by failing to check whether a file is not a directory, on top of checking if the file exists. Regardless, if you think that this doesn't cause any regressions, I'll merge it.

Reply:
I have a tester currently using this change as a 'daily driver' of sorts. It's not ideal but it should help catch any obvious regressions at the very least.

Reply:
Also - the empty string here actually renders the image suffix checker completely useless, since the code checks the ending of the path it gets from the cache against the suffixes list, and literally any call to `QString::endsWith("")` will return `true`. I removed an identical empty string from the audio suffix checker in https://github.com/AttorneyOnline/AO2-Client/pull/571, but evidently neglected to remove it from here as well.